templates/pytorch/nightly/fs-transformer.libsonnet

```diff
@@ -125,11 +138,41 @@ local tpus = import "../../tpus.libsonnet";
       },
     },
   },
+  local fastconfig = base.Convergence {
+    paramsOverride: {
+      maxEpoch: 25,
+      trainSubset: "train",
+      validSubset: "valid",
+      inputShapes: "256x64 512x32 640x16",
+      logSteps: 200,
+    },
+    command+: [
+      "bleu=`fairseq-generate",
+      "    /datasets/wmt18_en_de_bpej32k",
+      "    --remove-bpe --lenpen 0.6 --beam 4",
+      "    --path /tmp/checkpoints/checkpoint25.pt",
+      "    --skip-invalid-size-inputs-valid-test | grep BLEU",
+      "    | grep -v loadi | tail -1 | cut -d '=' -f 3| cut -d'.' -f 1`",
+      "echo 'BLEU score is' $bleu",
+      "test $bleu -gt 28",
+    ],
+    regressionTestConfig+: {
+      metric_success_conditions+: {
+        "wps": {
```

Comment:
is this the correct metric name?

Reply:
metric name should be your tensorboard tag + the aggregation strategy. The default aggregation strategy for all metrics in xl-ml-test is `final`: https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/templates/base.libsonnet#L38

So your metric would be tag_final or 'wps_final' if that's the tag name in tensorboard

Reply:
The naming convention for these keys is discussed [here](https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/metrics_handler/README.md#regression_test_config)